tests/scripts/minikube.sh

```diff
@@ -16,6 +16,91 @@ EOF
     minikube start --memory=3000 --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
 }
 
+# workaround for kube-dns CrashLoopBackOff issue with RBAC enabled
+#issue https://github.com/kubernetes/minikube/issues/1734 and https://github.com/kubernetes/minikube/issues/1722
+enable_roles_for_RBAC() {
+cat <<EOF | kubectl create -f -
+# Wide open access to the cluster (mostly for kubelet)
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: cluster-writer
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+  - nonResourceURLs: ["*"]
+    verbs: ["*"]
+
```

Comment:
I would remove those empty lines in between the objects.

Reply:
done